pdm/cli/utils.py

```diff
@@ -40,19 +38,21 @@ def _format_action(self, action: Action) -> str:
 
         # no help; start on same line and add a final newline
         if not action.help:
-            tup = self._current_indent, "", action_header
-            action_header = "%*s%s\n" % tup
+            action_header = "%*s%s\n" % (self._current_indent, "", action_header)
 
         # short action name; start on the same line and pad two spaces
         elif len(action_header) <= action_width:
-            tup = self._current_indent, "", action_width, action_header
-            action_header = "%*s%-*s  " % tup
+            action_header = "%*s%-*s  " % (
+                self._current_indent,
+                "",
+                action_width,
+                action_header,
+            )
```

Comment:
not entirely sure why mypy had a problem with re-aliasing the 'tup' variable, given it's scoped to the `if` blocks...

in any case, i'd appreciate it if someone could walk me through what these c-string style formatting strings are doing. How would people feel about updating these to f-strings?